yjit.rb

```diff
@@ -286,6 +286,7 @@ def _print_stats(out: $stderr) # :nodoc:
       out.puts "bindings_set:          " + format_number(13, stats[:binding_set])
       out.puts "compilation_failure:   " + format_number(13, compilation_failure) if compilation_failure != 0
       out.puts "compiled_iseq_count:   " + format_number(13, stats[:compiled_iseq_count])
+      out.puts "compiled_iseq_entry:   " + format_number(13, stats[:compiled_iseq_entry])
```

Comment:
```suggestion
      out.puts "compiled_iseq_entry:   " + format_number(13, stats[:compiled_iseq_entry])
      out.puts "compiled_iseq_count:   " + format_number(13, stats[:compiled_iseq_count])
```

It seems nicer to have them swapped to make it the later, the larger. It's consistent with the other in stats.rs as well.